tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/TlaType1.scala

```diff
@@ -36,15 +86,68 @@ case class StrT1() extends TlaType1 {
   */
 case class ConstT1(name: String) extends TlaType1 {
   override def toString: String = name
+
+  override def usedNames: Set[Int] = Set.empty
+
+  override def varInterval: (Int, Int) = (0, 0)
 }
 
 /**
-  * A type variable.
+  * A type variable. Instead of using strings for names, we are just using integers, which makes it easier
+  * to process them. To make vars user-friendly, we assign the names a..z to the numbers 0..25.
+  * The rest are called a27, a28, etc.
   *
-  * @param name a variable name that should be assigned a type
+  * @param no the variable number
```

Comment:
`no` reads as "no" as in the opposite of "yes" to me. I suggest using `n`, `num`, or `number`.